install/Dockerfile.cniserver

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:16.04
+
+RUN apt-get update \
+    && apt-get install -y\
+        net-tools \
+        iptables \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY bin/cniserver /var/libnetwork/bin/
```

Comment:
Shouldnt this be ../bin/ (this Dockerfile is in install directory

Reply:
Since its out of build context I have script to copy to the install. will rectify this later.